src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,184 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.InvalidUserDataException;
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String DEFAULT_LOGIN = "defaultLogin";
+    private static final String DEFAULT_PASSWORD = "defaultPassword";
+    private static final Integer DEFAULT_AGE = 36;
+    private static final String LESS_THAN_6_CHARACTERS = "short";
+    private static final String EMPTY_STRING = "";
+    private static final String NULL_STRING = null;
+    private static final Integer NULL_INTEGER = null;
+    private static final Integer ZERO_AGE = 0;
+    private static final Integer UNDERAGE = 17;
+    private static final Integer AGE_EIGHTEEN = 18;
+    private static final Integer NEGATIVE_AGE = -1;
+    private static final String SIX_CHARACTER_STRING = "six123";
+    private RegistrationServiceImpl registrationService;
+    private StorageDao storageDao;
+    private User user;
+
+    @BeforeEach
+    void setUp() {
+        storageDao = new StorageDaoImpl();
+        registrationService = new RegistrationServiceImpl(storageDao);
+        user = new User();
+        user.setLogin(DEFAULT_LOGIN);
+        user.setPassword(DEFAULT_PASSWORD);
+        user.setAge(DEFAULT_AGE);
+    }
+
+    @ AfterEach
+    void tearDown() {
+        storageDao.clear();
+    }
+
+    @Test
+    void checkLengthOfUsersLogin_Ok() {
+        user.setLogin(DEFAULT_LOGIN);
+        registrationService.register(user);
+        assertTrue(registrationService.isSuccesfullyRegistered(user));
+    }
+
+    @Test
+    void checkLengthOfUsersPassword_Ok() {
+        user.setPassword(DEFAULT_PASSWORD);
+        registrationService.register(user);
+        assertTrue(registrationService.isSuccesfullyRegistered(user));
```

Comment:
Similar to the login length, this test does not cover the edge cases for password length. You should include tests for passwords that are exactly at the minimum length boundary and one character shorter to confirm the correct behavior.